Math.md

```diff
@@ -0,0 +1,96 @@
+##Float and Double
+You've already learned about the String and Int types; Strings are values like "hello" or "eat your vegetables", and Ints are whole number values like 34 and 9561.  
+
+Strings may form the backbone of displaying instructions and values to the user, but numbers (and especially mathematical operations on those numbers) are necessary for actually drawing that text to the screen, as well as pretty much every thing the operating system and your applications do, from computing how many days are left in a subscripton to centering a button in window.  
+
+Another kind of number are Float and Double, which represent decimal values such as 0.5 or 3.14159.  The difference between Float and Double is magnitude and precision: Floats are represented in 32 bits and Doubles are represented in 64 bits, so Doubles have the potential for greater magnitude and precision.
+
+You can recognize Floats and Doubles by value because they will have a decimal point; the Swift compiler will interpret any number with a decimal point as a Double unless you specify that it should be recognized as a Float.  Any number without a decimal point will be interpreted by the compiler as an Int.
+
```

Comment:
I really like this.

I would like to add code showcasing this right below this paragraph like so:

```swift
let intNumber = 50

// intNumber is a variable of type Int with a value of 50


let doubleNumber = 3.225

// doubleNumber is a constant of type Double with a value of 3.225
```